docs/excel/excel-add-ins-ranges-advanced.md

```diff
@@ -83,7 +83,7 @@ getSpecialCellsOrNullObject(cellType: Excel.SpecialCellType, cellValueType?: Exc
 The following example uses the `getSpecialCells` method. About this code, note:
 
 - It limits the part of the sheet that needs to be searched by first calling `Worksheet.getUsedRange` and calling `getSpecialCells` for only that range.
-- It passes as a parameter to `getSpecialCells` the string version of a value from the `Excel.SpecialCellType` enum. Some of the other values that could be passed instead are "Blanks" for empty cells, "Constants" for cells with literal values instead of formulas, and "SameConditionalFormat" for cells that have the same conditional formatting as the first cell in the `usedRange`. The first cell is the upper leftmost cell. For a complete list of the values in the enum, see [beta office.d.ts](https://appsforoffice.microsoft.com/lib/beta/hosted/office.d.ts).
+- It passes as a parameter to `getSpecialCells` the string version of a value from the `Excel.SpecialCellType` enum. Some of the other values that could be passed instead are `blanks` for empty cells, `constants` for cells with literal values instead of formulas, and `sameConditionalFormat` for cells that have the same conditional formatting as the first cell in the `usedRange`. The first cell is the upper leftmost cell. For a complete list of the values in the enum, see [beta office.d.ts](https://appsforoffice.microsoft.com/lib/beta/hosted/office.d.ts).
```

Comment:
"It passes as a parameter to `getSpecialCells` the string version of a value from the `Excel.SpecialCellType` enum." -- does this correctly describe what the example does?  Seems like it's now using the enum to specify the parameter value (not "the string version of a value from the enum").

Reply:
I just removed this bullet point. I added text to say the example is getting formulas and we go over the enum later in the article.